src/app/components/PriceChart.tsx

```diff
@@ -316,3 +354,126 @@ export function PriceChart() {
     </>
   );
 }
+
+export function PairInfoTab() {
+  const t = useTranslations();
+  const { pairsList } = useAppSelector((state) => state.rewardSlice);
+  const selectedPairAddress = useAppSelector(
+    (state) => state.pairSelector.address
+  );
+  const pairInfo = pairsList.find(
+    (pairInfo) => pairInfo.address === selectedPairAddress
+  );
+
+  let name, address, orderReceiptAddress, token1, token2;
+  if (pairInfo) {
+    ({ name, address, orderReceiptAddress, token1, token2 } = pairInfo);
+  } else {
+    return "Selected pair not found in pairsList";
+  }
+
+  const shortenAddress = (address: any) => {
+    if (!address) return "";
+    return `${address.slice(0, 40)}...`;
+  };
```

Comment:
Please use [this function](https://github.com/DeXter-on-Radix/website/blob/29a69e7179492e98bfbab1185119296ffb7f8fea/src/app/utils.ts#L182-L196) instead :)